public/javascript/index.js

```diff
@@ -1,5 +1,8 @@
 const charactersAPI = new APIHandler('http://localhost:8000');
 
+
+
+
 window.addEventListener('load', () => {
```

Comment:
Here you are missing calling the API functions inside of the eventListeners and doing the respective DOM manipulation to display the responses from the API in our index.html file